src/transformers/models/blip_2/modeling_blip_2.py

```diff
@@ -304,7 +401,13 @@ class Blip2PreTrainedModel(PreTrainedModel):
     config_class = Blip2Config
     base_model_prefix = "blip"
     supports_gradient_checkpointing = True
-    _no_split_modules = ["Blip2Attention", "T5Block", "OPTDecoderLayer"]
+    _no_split_modules = [
```

Comment:
@NielsRogge I updated `_no_split_modules`, i hope this will fix the slow multi-gpu tests that were failing.